packages/flutter_tools/test/general.shard/ios/xcodeproj_test.dart

```diff
@@ -616,9 +616,11 @@ Information about project "Runner":
   });
 
   testWithoutContext('expected scheme for non-flavored build is Runner', () {
-    expect(XcodeProjectInfo.expectedSchemeFor(BuildInfo.debug), 'Runner');
-    expect(XcodeProjectInfo.expectedSchemeFor(BuildInfo.profile), 'Runner');
-    expect(XcodeProjectInfo.expectedSchemeFor(BuildInfo.release), 'Runner');
+    final XcodeProjectInfo info = XcodeProjectInfo(<String>['Runner'], <String>['Debug', 'Release', 'Runner'], <String>['Runner'], logger);
+
+    expect(info.schemeFor(hostAppProjectName: 'Runner', flavor: BuildInfo.debug.flavor), 'Runner');
+    expect(info.schemeFor(hostAppProjectName: 'Runner', flavor: BuildInfo.profile.flavor), 'Runner');
+    expect(info.schemeFor(hostAppProjectName: 'Runner', flavor: BuildInfo.release.flavor), 'Runner');
```

Comment:
`BuildInfo.debug.flavor` is null in all these cases, right? It's not set?  So these are all the same:

```dart
    expect(info.schemeFor(hostAppProjectName: 'Runner'), 'Runner');
    expect(info.schemeFor(hostAppProjectName: 'Runner'), 'Runner');
    expect(info.schemeFor(hostAppProjectName: 'Runner'), 'Runner');
```

Seems like a good place to check casing:

```suggestion
    expect(info.schemeFor(hostAppProjectName: 'Runner'), 'Runner');
    expect(info.schemeFor(hostAppProjectName: 'runner'), 'Runner');
    expect(info.schemeFor(hostAppProjectName: 'RUNNER'), 'Runner');
```